app/models/concerns/capital.rb

```diff
@@ -4,6 +4,10 @@ def positive?
   end
 
   def amount_attributes
-    attributes.except("id", "legal_aid_application_id", "updated_at", "created_at", "none_selected", "no_account_selected")
+    shared_attributes.except("partner_offline_current_accounts", "partner_offline_savings_accounts")
```

Comment:
The next ticket may need to extract only the partner fields, so I separated this rather than merging the list into a single entity and then extracting again immediately